test/visibility/except/exceptModuleItself.good

```diff
@@ -1,2 +1,2 @@
 exceptModuleItself.chpl:6: In function 'main':
-exceptModuleItself.chpl:7: error: Bad identifier in 'except' clause, no known 'Toolbox' defined in 'Toolbox'
+exceptModuleItself.chpl:7: error: cannot use 'except' clause with symbol 'Toolbox' as it is not defined in 'Toolbox'
```

Comment:
I skipped over this one the last time I saw it, but shouldn't "as if..." be set off by a comma?